src/Filesystem/File.php

```diff
@@ -372,11 +372,11 @@ public function mime()
     /**
      * Returns the file's last modification time
      *
-     * @param string $format
-     * @param string|null $handler date or strftime
+     * @param string|array|null $format
```

Comment:
```suggestion
     * @param string|array|\IntlDateFormatter|null $format
```

Or without `array` if we decide to remove that option.